src/App.tsx

```diff
@@ -3,83 +3,87 @@ import 'bulma/css/bulma.css';
 import './App.scss';
 
 export const goods = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
+  { id: 1, name: 'Dumplings' },
+  { id: 2, name: 'Carrot' },
+  { id: 3, name: 'Eggs' },
+  { id: 4, name: 'Ice cream' },
+  { id: 5, name: 'Apple' },
+  { id: 6, name: 'Bread' },
+  { id: 7, name: 'Fish' },
+  { id: 8, name: 'Honey' },
+  { id: 9, name: 'Jam' },
+  { id: 10, name: 'Garlic' },
 ];
```

Comment:
key needs a unique identificator(but not the index) but you don't have to create id for it specifically because you already have a unique identificator in this task = name it's unique and you can use it as a key.

Reply:
Good job for removing id, but I also meant you don't even need to create an array of objects just use already given array. But whatever floats your boat I guess.

Reply:
check @nikitaovch99 comment
you don't need create an array of objects here, it's redundant